### PR TITLE
Fix special coverage migration save

### PR DESCRIPTION
**Pull request by mparent61**
v0.7.3  Fix special coverage migration to grab model directly so that custom SpecialCoverage.save() method is called and triggers _save_percolator.

A model obtained from apps.get_model() calls the generic save(), and *does not* call a custom save() method like SpecialCoverage.save(). This surprised me, take heed when using app.get_models() in your migration scripts. 

https://realpython.com/blog/python/data-migrations/#comment-1843026722

### CODE REVIEW

**Comment by mparent61** [comment]
@benghaziboy @MichaelButkovic

**Comment by MichaelButkovic** [comment]
huh, very weird

**Comment by MichaelButkovic** [comment]
LGTM

File: bulbs/special_coverage/migrations/0009_auto_20160208_1540.py [hunk 1]
```diff
@@ -1,13 +1,14 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
-from django.db import migrations, models
+from django.db import migrations
+
+from bulbs.special_coverage.models import SpecialCoverage
```
Comment (benghaziboy) [inline bulbs/special_coverage/migrations/0009_auto_20160208_1540.py]: Have you re-run this locally? 

Migrations can have dependency issues if not explicit, could be wrong here.

**Comment by benghaziboy** [comment]
provided this has run,

LGTM

**Comment by mparent61** [comment]
Yeah runs locally.
I think the dependency issues involve cross-app dependencies. Also by not using get_model() you can run into trouble if your migration also modifies schema.

**Pull request merged**